resources/views/users/edit-add.blade.php

```diff
@@ -79,6 +79,7 @@
                                 <input type="file" name="avatar">
                             </div>
 
+                            @if (Auth::user()->role_id == "1")
```

Comment:
I don't really like using the hard-coded role ID.  Also, I believe the role_id **should** be numerical, not a string.